### PR TITLE
Added instructions for BigQuery integration to Flink via Aiven Console

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
# What changed, and why it matters

Added instructions on integrating Aiven for Apache Flink with BigQuery as SINK using Aiven Console.

### CODE REVIEW

File: docs/products/flink/howto/connect-bigquery.rst [hunk 1]
```diff
@@ -205,3 +205,26 @@ Following is an  example of a Google BigQuery SINK table:
...
+If you're using Google BigQuery for your data storage and analysis, you can seamlessly integrate it as a sink for Aiven for Apache Flink streams. To achieve this via the `Aiven Console <https://console.aiven.io/>`_, follow these steps:
+
+1. Log in to `Aiven Console <https://console.aiven.io/>`_ and choose your project. 
+2. From the **Services** page, you can either :doc:`create a new Aiven for Apache Flink </docs/platform/howto/create_new_service>` service or select an existing service.
+3. Next, configure Google BigQuery service integration endpoint:
+  
+   * Navigate to the **Projects** screen where all the services are listed. 
+   * From the left sidebar, select **Integration endpoints**. 
+   * Select **Google Cloud BigQuery** from the list, and then select **Add new endpoint** or **Create new**. 
+   * Enter an *Endpoint name*, *GCP Project ID*, *Google Service Account Credentials*, and select **Create**.
```
Comment (juha-aiven) [inline docs/products/flink/howto/connect-bigquery.rst:220]: in the Aiven Client instructions there's more detail on these, especially on "Google Service Account Credentials". Maybe have those instruction centralized or have them also here?

File: docs/products/flink/howto/ext-kafka-flink-integration.rst
Comment (juha-aiven) [inline docs/products/flink/howto/ext-kafka-flink-integration.rst:1]: This change should probably be in its own PR, it's about external Kafka, not about BQ.

**Review by juha-aiven — CHANGES_REQUESTED** [review]
Looks good, except for the two comments I left.

File: docs/products/flink/howto/connect-bigquery.rst (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/flink/howto/connect-bigquery.rst:220]: > in the Aiven Client instructions there's more detail on these, especially on "Google Service Account Credentials". Maybe have those instruction centralized or have them also here?

With the console's new side menu design, it's beneficial to have these instructions right there when users are doing tasks. This way, they don't have to keep going between different documents and screens. Having a link that takes them to another article for just four steps is unnecessary. The Google BigQUery integration doc in a common location (mostly Service integrations) is also still WIP. But once this is complete, I can update this article to include the link.

File: docs/products/flink/howto/ext-kafka-flink-integration.rst
Comment (harshini-rangaswamy) [reply docs/products/flink/howto/ext-kafka-flink-integration.rst:1]: > This change should probably be in its own PR, it's about external Kafka, not about BQ.

Combining both changes in one pull request (PR) is more efficient for documentation purposes. Since these changes are related to the new UI design in the console for external integration (which was different previously), it makes sense to bundle them together. Please feel free to elaborate on what value add separate PRs would bring to the documentation process specifically.

**Comment by harshini-rangaswamy** [comment]
> Looks good, except for the two comments I left.

Thanks, @juha-aiven! If there are no changes to the technical accuracy of the docs, we can merge this.

File: docs/products/flink/howto/ext-kafka-flink-integration.rst
Comment (juha-aiven) [reply docs/products/flink/howto/ext-kafka-flink-integration.rst:1]: > Please feel free to elaborate on what value add separate PRs would bring to the documentation process specifically.

The PR description and topic should tell what the PR is about. Now the Kafka integration documentation change comes as a surprise since it's not mentioned in the PR description. It's neither mentioned in the commit message.

I'll merge this, it's a small detail.

Comment (harshini-rangaswamy) [reply docs/products/flink/howto/ext-kafka-flink-integration.rst:1]: > > Please feel free to elaborate on what value add separate PRs would bring to the documentation process specifically.
> 
> The PR description and topic should tell what the PR is about. Now the Kafka integration documentation change comes as a surprise since it's not mentioned in the PR description. It's neither mentioned in the commit message.
> 
> I'll merge this, it's a small detail.

Understood! I will keep that in mind and make sure to include these details in the description.

**Review by juha-aiven — APPROVED** [review]
LGTM
